src/trade.js

```diff
@@ -284,6 +295,7 @@ class Trade {
       // Only consider transactions that are complete or that we're still
       // expecting payment for:
       return [
+        'awaiting_reference_number',
```

Comment:
Is this specific to Unocoin? If so, I don't think it should be in this list. This data should probably be implemented by descendent classes.